### ISSUE TITLE
Remove mentions of WAMP testing in docs, remove any remaing code

### ISSUE THREAD

**Issue by oberstet** [tavendo/AutobahnTestSuite#42]
WAMP testing won't be in AutobahnTestsuite for various reasons.

**Comment by theduke** [comment]
Should really be removed from the website especially.

**Comment by oberstet** [comment]
@theduke Yes, there is no public WAMP testsuite, which is the reason for this issue.

**Comment by kskyten** [comment]
Is this the reason why `wstest -d -m wampclient -w <Your Server IP:Port>` won't work in a pip install of the testsuite unlike indicated in [the docs](https://github.com/crossbario/autobahn-testsuite/blob/master/doc/usage.rst#mode-wampserverwampclient)?
